app/views/spree/admin/shared/_order_tabs.html.haml

```diff
@@ -0,0 +1,63 @@
+- content_for :page_title do
+  = t(:order)
+  \#
+  = @order.number
+
+- if @order.bill_address.present?
+  = @order.bill_address.firstname
+  = @order.bill_address.lastname
+  \-
+
+- content_for :sidebar_title do
+  = t(:order_information)
+
+- content_for :sidebar do
+  %header#order_tab_summary
+    %dl.additional-info
+      %dt#order_status
+        = t(:status)
+      %dd
+        %span{:class => "state #{@order.state}"}
```

Comment:
Hey @HugsDaniel it's just these classes with string interpolation that are not showing up on the html.
this is the only way I managed to make it work 
```suggestion
        - span_classes = "state #{@order.state}"
        %span{class: span_classes}
```

if no one can find a better option, I'd go ahead with this...

Reply:
WTF? The result should be exactly the same. Is it a caching issue?

Reply:
it's something on haml I believe because with the initially solution, it's not just the interpolation that is broken, the result of the original code is a span without the class tag all together. if I remove the interpolation part I get a span with class="state".

Reply:
Yeah, it seems to be an outdated HAML version issue.

Reply:
Okay, let's go with your solution then and let's check if we can update HAML.

Reply:
Ah, we have to wait until we can upgrade Rails as well: https://github.com/openfoodfoundation/openfoodnetwork/pull/2596#issuecomment-416468596

Reply:
I don't think this deserves too much attention. I vote to use this workaround anyway.

Reply:
That's the only combination I didn't try haha it should be ready to test now !